### ISSUE TITLE
Missing alt-text on images

### ISSUE THREAD

**Issue by phucduong86** [shiffman/shiffman.net#79]
Multiple elements:
**Homepage:**
Missing text alternative on all images
```
<img src="images/ct-logo.png" width="200px">
<img class="rainbow" src="images/train.png" width="300px">
<img class="book" src="images/book-lp.png" width="131px">
<img class="book" src="images/book-noc.png" width="131px">
```

**Books**
```
<img src="/images/nav-btn.svg">
<img class="book medium-hide" src="/images/book-lp.png" width="131px">
<img class="book medium-hide" src="/images/book-noc.png" width="131px">
```
**Videos:**
```
<img src="/images/nav-btn.svg">
<img src="/images/icon-youtube.png" class="icon">
```

**Learning**
```
<img src="/images/nav-btn.svg">
<img src="/images/atoz.png" width="800">
<img src="/images/kinect.png" width="800">
```

**a2z**
Frames element on youtube videos need tittle
